packages/SwingSet/tools/bundleTool.js

```diff
@@ -31,7 +31,12 @@ const providedCaches = new Map();
  * destination, return it.
  *
  * @param {string} dest
- * @param {{ format?: string, dev?: boolean }} options
+ * @param {object} options
+ * @param {string} [options.format]
+ * @param {boolean} [options.dev]
+ * @param {number} [options.byteLimit] - Maximum bundle size in bytes before
+ *   falling back to optimisation that sacrifices legibility. Defaults to
+ *   500,000 bytes.
```

Comment:
JSDoc has a defaults syntax. Also consider American spelling :)

```suggestion
 * @param {number} [options.byteLimit=500_000] - Maximum bundle size in bytes before
 *   falling back to optimization that sacrifices legibility.
```

Reply:
I had to put `'jsdoc/no-defaults': 'off'` in `eslintrc.config.mjs`, but I think that's a worthwhile change.  Oh, and thanks for reminding me that our lingua franca is American. ;P